rviz_default_plugins/package.xml

```diff
@@ -64,6 +64,8 @@
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>rviz_rendering_tests</test_depend>
   <test_depend>rviz_visual_testing_framework</test_depend>
+  <test_depend>image_transport_plugins</test_depend>
```

Comment:
`image_transport_plugins` is not a [ROS 2 core package](https://github.com/ros2/ros2/blob/rolling/ros2.repos), we should remove this from here